rest/specs/src/main/filtered-resources/openapi/openapi.yaml

```diff
@@ -0,0 +1,306 @@
+# /**
+#  * Copyright (c) Connexta
+#  *
+#  * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+#  * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+#  * the License, or any later version.
+#  *
+#  * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+#  * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+#  * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+#  * License is distributed along with this program and can be found at
+#  * <http://www.gnu.org/licenses/lgpl.html>.
+#  */
+openapi: 3.0.2
+
+info:
+  version: ${project.version}
+  title: Transformation API Specification
+  description: >
+    The Transformation API allows products to be transformed into discovery metadata and other supporting products.
+  contact:
+    name: Connexta
+  license:
+    name: GNU Lesser General Public License
+    url: 'http://www.gnu.org/licenses/lgpl.html'
+
+servers:
+  - url: ${rest.server.url}
+
+paths:
+  /transform:
+    parameters:
+      - $ref: '#/components/parameters/AcceptVersion'
+    post:
+      summary: Transformation request
+      description: A request to transform product into discovery metadata and other supporting products.
+      operationId: transform
+      tags:
+        - transform
+      requestBody:
+        $ref: '#/components/requestBodies/TransformRequest'
+      responses:
+        '202':
+          $ref: '#/components/responses/Accepted'
+        '400':
+          $ref: '#/components/responses/BadRequest'
+        '401':
+          $ref: '#/components/responses/Unauthorized'
+        '403':
+          $ref: '#/components/responses/Forbidden'
+        '500':
+          $ref: '#/components/responses/InternalServerError'
+        '501':
+          $ref: '#/components/responses/NotImplemented'
+        default:
+          $ref: '#/components/responses/DefaultError'
+
+components:
+  requestBodies:
+    TransformRequest:
+      description: A request to transform product into discovery metadata and other supporting products.
+      required: true
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/TransformRequest'
+  responses:
+    Accepted:
+      description: The transformation request was accepted for processing.
+      headers:
+        'Content-Version':
+          $ref: '#/components/headers/ContentVersion'
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/TransformResponse'
+    BadRequest:
+      description: The client message could not understood by the server due to invalid format or syntax.
```

Comment:
@figliold - ✏️ Minor wording fix: "...could not *be* understood by the server..."